<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterMainWindow</class>
 <widget class="QMainWindow" name="RegisterMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1326</width>
    <height>605</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1326</width>
      <height>605</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="registerLittleFrame">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>50</y>
       <width>480</width>
       <height>530</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>欢迎注册</string>
      </property>
     </widget>
     <widget class="QLabel" name="accountLabel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>已有账号？</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginPushButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>68</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 255);
font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Usernamelabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>124</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>用  户  名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="UsernamelineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwprdlineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="passwordlabel_">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>密       码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="identityCodelineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>245</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="partmentlabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>身份证号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phonenumberlineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>315</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="phonenumberlabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>320</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>手  机  号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EmaillineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>380</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Emaillabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>385</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>邮       箱</string>
      </property>
     </widget>
     <widget class="QPushButton" name="registerpushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>450</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(5, 155, 236);
border-radius:20px;</string>
      </property>
      <property name="text">
       <string>注 册</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="registercheckBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>500</y>
        <width>331</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>阅读并接受《跃渊用户协议》及《跃渊隐私权保护声明》</string>
      </property>
     </widget>
     <widget class="QLabel" name="usernameTiplabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>154</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="passwordTiplabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="identityCodeTiplabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>288</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="phoneNumberTiplabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>355</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="emailTiplabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>420</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="titlebarframe">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1326</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 242, 246);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="closePushButton">
      <property name="geometry">
       <rect>
        <x>1300</x>
        <y>7</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/photo01/image/LoginAndRegister/toast_close_black.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>311</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>跃渊文件管理系统</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
